<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_4">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="ProgramerBox">
          <property name="locale">
           <locale language="Nepali" country="Nepal"/>
          </property>
          <property name="title">
           <string>tr(programer)</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QComboBox" name="SelectProgrammer">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="PortBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Porв</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QComboBox" name="SelectPort">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <item>
              <property name="text">
               <string>Новий елемент</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Новий елемент</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Новий елемент</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Новий елемент</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="Step1Box">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Step1</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="ButtonFuse">
           <property name="text">
            <string>Fuse</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="Step2Box">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Step2</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QPushButton" name="ButtonDiscovery">
           <property name="text">
            <string>Discovery</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="ButtonNanitInfo">
           <property name="text">
            <string>Nanit</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="BootLoaderCheck">
           <property name="text">
            <string>Bootloader</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="Step3Box">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Step3</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="ButtonConfig">
           <property name="text">
            <string>Config</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="Step4Box">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Step4</string>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="ButtonSerial">
           <property name="text">
            <string>SERIAL</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="InputSerial">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="ExtraBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Extra</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="ButtonWipe">
           <property name="text">
            <string>Wipe</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="ButtonClear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonHelp">
        <property name="text">
         <string>help</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="Text"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuUpdate">
     <property name="title">
      <string>Update</string>
     </property>
     <addaction name="actionDiscovery"/>
     <addaction name="actionNanit"/>
     <addaction name="actionConfig"/>
     <addaction name="actionAll"/>
    </widget>
    <addaction name="actionCheckUpdate"/>
    <addaction name="menuUpdate"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionCheckUpdate">
   <property name="text">
    <string>CheckUpdate</string>
   </property>
  </action>
  <action name="actionDiscovery">
   <property name="text">
    <string>Discovery</string>
   </property>
  </action>
  <action name="actionNanit">
   <property name="text">
    <string>Nanit</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>SelectProgrammer</tabstop>
  <tabstop>SelectPort</tabstop>
  <tabstop>ButtonFuse</tabstop>
  <tabstop>BootLoaderCheck</tabstop>
  <tabstop>ButtonDiscovery</tabstop>
  <tabstop>ButtonNanitInfo</tabstop>
  <tabstop>ButtonConfig</tabstop>
  <tabstop>InputSerial</tabstop>
  <tabstop>ButtonSerial</tabstop>
  <tabstop>Text</tabstop>
  <tabstop>ButtonWipe</tabstop>
  <tabstop>ButtonClear</tabstop>
  <tabstop>ButtonHelp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
